src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -24,6 +25,7 @@ private Bugsnag() {}
      */
     public static Client init(Context androidContext) {
         client = new Client(androidContext);
+        configureClientObservers();
```

Comment:
Could (should?) this be moved into the constructor for the `Client` object itself? As far as I remember, a supported way of using Bugsnag on Android is by instantiating a `Client` object directly and not using the `Bugsnag` static class.